src/e3/anod/spec.py

```diff
@@ -205,6 +206,20 @@ def __init__(
         # Hold the result of the pre function
         self._pre: Optional[Dict[str, Any]] = None
 
+    def _fetch_attr(self, name: str, default_value: Any) -> Any:
```

Comment:
Do you think you can move this function outside this class?

I'd like to avoid polluting specs namespace. Btw thank you for taking the time to find this much better solution!

Reply:
Sure!

Ah yes, this should be done, now, thanks.